elixir/apps/web/lib/web/live/clients/index.ex

```diff
@@ -5,6 +5,7 @@ defmodule Web.Clients.Index do
 
   def mount(_params, _session, socket) do
     with {:ok, clients} <- Clients.list_clients(socket.assigns.subject, preload: :actor) do
+      clients = Enum.sort_by(clients, & &1.online?, :desc)
```

Comment:
This line will go away once we have pagination and filters so I did not cover it with tests.